main.go

```diff
@@ -103,9 +106,15 @@ func buildGraph() bool {
 		clearLine()
 		fmt.Printf("Page %d analyzed...\r", page)
 	}
+
+	d3Data.RequestsUsed = RateLimitSpecs.Limit - RateLimitSpecs.Remaining
+	d3Data.MaxRequests = RateLimitSpecs.Limit
+	d3Data.LastUpdate = time.Now().Format(time.RFC822Z)
+	d3Data.RefreshInterval = nextRefresh
+
 	// Output to memory
 	MarshalToMemory(d3Data)
```

Comment:
`MarshalToMemory` could split to `MarshalD3ToMemory` (current function body) & `MarshalDashboardToMemory` (same body, but output to the dedicated struct cited above.
The `MarshalToFile` method can be considered deprecated and will be removed, so no need to update that.